packages/tsconfig-reference/copy/en/options/noFallthroughCasesInSwitch.md

```diff
@@ -4,7 +4,7 @@ oneline: "Enable error reporting for fallthrough cases in switch statements."
 ---
 
 Report errors for fallthrough cases in switch statements.
-Ensures that any non-empty case inside a switch statement includes either `break` or `return`.
+Ensures that any non-empty case inside a switch statement includes either `break`, `return` or `throw`.
```

Comment:
```suggestion
Ensures that any non-empty case inside a switch statement includes either `break`, `return`, or `throw`.
```

Reply:
(IIRC we try to be consistent about using the oxford comma, but, willing to be wrong.)